tsconfig.json

```
@@ -7,6 +7,8 @@
     "declaration": true,
     "strict": true
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["src/**/*.ts", "src/*.ts"],
```

```suggestion
  "include": ["src/**/*.ts"],
```
The globe pattern `.../**/*...` already covers `.../*...`